controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -990,6 +1006,21 @@ private Slime toSlime(Tenant tenant, HttpRequest request, boolean listApplicatio
         return slime;
     }
 
+    private Slime toSlime(URI propertyUri, URI contactsUri, URI issueCreationUri, List<? extends List<? extends User>> contacts) {
+        Slime slime = new Slime();
+        Cursor root = slime.setObject();
+        root.setString("propertyUri", propertyUri.toString());
+        root.setString("contactsUri", contactsUri.toString());
+        root.setString("issueCreationUri", issueCreationUri.toString());
```

Comment:
"Uri" -> "Url" in all these fields. This is how we name other URL fields as well. We sometimes confuse the terms as we always use `URI` to represent URLs as the `URL` class has strange behavior.

Reply:
Sure. And you're right, of course. Why not be specific when you can?